public_html/admin/language/english/extension/extensions.xml

```diff
@@ -409,5 +413,20 @@
 		<key>button_get_support</key>
 		<value><![CDATA[Get Support]]></value>
 	</definition>
-
+	<definition>
+		<key>text_min_volume</key>
+		<value><![CDATA[Restriction Minimal Volume of Parcel (cubic %s)]]></value>
+	</definition>
+	<definition>
+		<key>text_max_volume</key>
+		<value><![CDATA[Restriction Maximal Volume of Parcel (cubic %s)]]></value>
```

Comment:
Maximum order parcel size volume in (cubic %s) <span class="help">Maximum value for shipment option to be available for customer. This value is based on combined products sizes in the order</span>

Reply:
Github hided the span tag. It is starting from < Maximum ...>